<launch>

  <!-- Include the shared resources launch file -->
  <include file="$(find omniroute_operation)/launch/shared_resources.launch" />

  <!-- Omniroute omniroute_controller interface node -->
  <node name="omniroute_controller_node" pkg="omniroute_operation" type="omniroute_controller" output="screen" />

</launch>